python/lsst/analysis/tools/tasks/limitingSurfBriAnalysis.py

```diff
@@ -0,0 +1,338 @@
+# This file is part of analysis_tools.
```

Comment:
Should this file have been replaced by the new one?

Reply:
Yes, forgot to add deleted files to the commit.  Done now.